<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PSE Simulation Interface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_run">
      <property name="text">
       <string>Run Simulations</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_stop">
      <property name="text">
       <string>Force Stop</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_">
       <attribute name="title">
        <string>Parameter Files</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QFrame" name="frame_params">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="3">
            <widget class="QPushButton" name="pushButton_outdir">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_outdir"/>
           </item>
           <item row="4" column="1">
            <widget class="QCheckBox" name="checkBox_video">
             <property name="text">
              <string>Generate Video</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="4">
            <widget class="QListWidget" name="listWidget"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_params"/>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="pushButton_params">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Parameter Folder</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Simulation Directory</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QCheckBox" name="checkBox_statistics">
             <property name="text">
              <string>Generate Statistics</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_settings">
       <attribute name="title">
        <string>Simulation Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Run Simulation Script</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit_sim">
          <property name="plainText">
           <string>julia --project scripts/run_from_xlsx.jl %parameterFile% </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Generate Statistics Script</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit_stats"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Run Simulation Script</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit_sim_video"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
